doc/howtos/rdtraining/12_sprinkles.rst

```diff
@@ -128,6 +128,10 @@ take extra options. An exhaustive list can be found in :ref:`reference/js/widget
     Tip: a simple example can be found
     `here <https://github.com/odoo/odoo/blob/0e12fa135882cd5095dbf15fe2f64231c6a84336/addons/account/views/account_bank_statement_views.xml#L136>`__.
 
+.. warning:: Same field multiple times in a view
+
+    Always add a field only **once** in a view. Adding it multiple times is simply not supported.
```

Comment:
I'd keep the warning but there are some exceptions to this: inline views and kanban/gantt?/etc views